### DISCUSSION TITLE
Install guide to set up stand alone MPC

### DISCUSSION THREAD

**Discussion by gnosys-ccoupal** [microsoft/PlanetaryComputer#48] [General]
Good day! Is there a guide to setting up a stand-alone MPC with the visualization interface, STAC server, etc? My use case has proprietary datasets and we would like to bring the functionality of MPC to these datasets. I see the GitHub repos, but note these are part of your CI/CD and I'd appreciate guidance on secrets, environmental variables, etc.

**Comment by TomAugspurger** [comment]
We unfortunately don't have anything like that today. We don't think we have the bandwidth to maintain that type of guide, especially since the technologies are moving pretty quickly.

So the open-source repositories (https://github.com/microsoft/planetary-computer-apis, https://github.com/microsoft/PlanetaryComputerDataCatalog/, https://github.com/microsoft/planetary-computer-hub) are the best we have at the moment.
